lib/modules/manager/ocb/extract.ts

```diff
@@ -0,0 +1,78 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { parseSingleYaml } from '../../../util/yaml';
+import { GoDatasource } from '../../datasource/go';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type Module, type OCBConfig, OCBConfigSchema } from './schema';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string,
+  _config?: ExtractConfig,
+): PackageFileContent | null {
+  let definition: OCBConfig | null = null;
+  try {
+    const yaml = parseSingleYaml(content);
+    const parsed = OCBConfigSchema.safeParse(yaml);
+    if (!parsed.success) {
+      logger.trace(
+        { packageFile, error: parsed.error },
+        'Failed to parse OCB schema',
+      );
+      return null;
+    }
+
+    definition = parsed.data;
+  } catch (error) {
+    logger.debug({ packageFile, error }, 'Failed to parse OCB file as YAML');
```

Comment:
```suggestion
    logger.debug({ packageFile, error }, 'Failed to parse OCB file (OpenTelemetryCollectorBuilder) as YAML');
```

Should we expand the abbrevation `OCB` in the debug log message? To me it's not clear what the abbrevation means, I guess: `OpenTelemetryCollectorBuilder`?

Reply:
```suggestion
    logger.debug({ packageFile, error }, 'OCB manager failed to parse file as YAML');
```
I find this rather verbose, but it make sense to give the user context where in the docs to search for OCB.

Reply:
Can you apply your suggestion? I don't have rights to do so. 😄